docs/markdown/snippets/rust_std_option.md

```diff
@@ -0,0 +1,5 @@
+## Rust now has a a standard option
+
+Rust calls these `editions`, but I've chosen to use `std`, as this is what
+meson uses for all other languages, even when those languages use a different
+term. Currently accepted values or 2015 and 2018
```

Comment:
Release note snippets should not be written in the first person. Instead they should say something like "but in Meson they are called ...".